package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mediterranea-marine-nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
```

Comment:
Can we configure eslint so it applies before doing a commit, and reject the commit otherwise?